hera_pspec/pspecdata.py

```diff
@@ -783,10 +807,36 @@ def set_weighting(self, data_weighting):
         Parameters
         ----------
         data_weighting : str
-            Type of data weightings. Options=['identity', 'iC']
+            Type of data weightings. Options=['identity', 'iC', 'sinc_downweight']
         """
         self.data_weighting = data_weighting
 
+    def set_r_param(self, key, r_params):
+        """
+        Set the weighting parameters for baseline at (dset,bl, pol)
+
+        Parameters
+        ----------
+        key: tuple (dset, bl, [pol]), where dset is the index of the dataset
+             bl is a 2-tuple
+             pol is an index of polarization of r_matrix.
+
+        r_params: dictionary with parameters for weighting matrix.
+                  Proper fields
+                  and formats depend on the mode of data_weighting.
+                data_weighting == 'sinc_downweight':
+                                dictionary with fields
+                                'filter_centers', list of floats (or float) specifying the centers of clean windows
+                                                  in units of 1/(channel index)
+                                'filter_widths', list of floats (or float) specifying the width of clean windows
```

Comment:
Isn't `channel_index` a dimensionless integer? So this just means the width is specified in an integer number of channels?

Reply:
It doesn't have to be an integer. I've made the language a little clearer and explicitly stated that fractional channel numbers can be used.